doc/design.md

```diff
@@ -201,7 +207,7 @@ Returns: Search results
     * tbd
 
 
-**`GET`** `/stats`
+**`GET`** `/api/<version>/stats`
```

Comment:
@a-ilango Sending data to the server should be POST instead of GET, and POST should return 201 on success. I agree with the English wording about getting a token but this kind of operation is done w/POST. Returning 401 UNAUTHORIZED on bad username or password seems reasonable

Reply:
Okay. I can change it back to POST. Why do we want to use "201 created"? Is it because we create a token based on user supplied data?

Reply:
Yes, exactly. Any time we are sending data to the server for processing or for storing in the database it should be in a POST or PUT body. In this case we should use POST